plans/001-big-blocks/tests/app-sync/run_seeds.go

```diff
@@ -63,7 +63,7 @@ func RunSeed(runenv *runtime.RunEnv, initCtx *run.InitContext) error {
 	}
 
 	initGenCh := make(chan string)
-	sub, err := syncclient.Subscribe(ctx, testkit.InitialGenenesisTopic, initGenCh)
+	sub, err := syncclient.Subscribe(ctx, sdk.InitialGenenesisTopic, initGenCh)
```

Comment:
oops 😅 
I suppose we need to change back to testkit to make it work, isn't it?

Reply:
😅